rr_arduino/rr_arduino.ino

```diff
@@ -174,3 +181,29 @@ void get_battery_state(long current_time){
     prev_time = millis();
   }
 }
+
+void raw_to_odom(double vel, int str_angle) {
```

Comment:
the steering angle is passed in degrees when it is recieved, you can grab it there and use that